transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_opportunity.sql

```diff
@@ -22,7 +23,8 @@ WITH freemium_opportunity_to_sync AS (
         ON customers_with_cloud_paid_subs.opportunity_external_id = opportunity.dwh_external_id__c
     LEFT JOIN {{ ref('account') }}
         ON customers_with_cloud_paid_subs.account_external_id = account.dwh_external_id__c
-    WHERE opportunity.id IS NULL
+        OR customers_with_cloud_paid_subs.domain = account.cbit__clearbitdomain__c
+    WHERE opportunity.id IS NULL AND account.dwh_external_id__c IS NOT NULL -- do not create opportunity if account does not exists
```

Comment:
Just checking to make sure we'll create an account if there isn't one...

Reply:
yes, following model will create a new account in case account and contact does not exist.
transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_account.sql

Reply:
sounds good, thanks!